core/app/[locale]/(default)/account/(tabs)/_components/modal.tsx

```diff
@@ -9,7 +9,7 @@ import { Button } from '~/components/ui/button';
 interface Props extends PropsWithChildren {
   actionHandler?: MouseEventHandler<HTMLButtonElement>;
   title: string;
-  trigger: React.ReactNode;
+  trigger?: React.ReactNode;
```

Comment:
Don't you get an error/warning if there's no trigger defined for a modal using radix?

Reply:
no, no errors